projects/plugins/jetpack/modules/search/instant-search/lib/hooks/use-photon.js

```diff
@@ -27,17 +27,31 @@ function stripQueryString( url ) {
  */
 export function usePhoton( initialSrc, width, height, isPhotonEnabled = true ) {
 	const [ src, setSrc ] = useState( null );
+	const initialSrcWithoutQueryString = stripQueryString( initialSrc );
+
+	// Photon only supports GIF, JPG and PNG
+	// @see https://developer.wordpress.com/docs/photon/
+	const supportedImageTypes = [ 'gif', 'jpg', 'png' ];
```

Comment:
should we worry about `.jpeg` extensions?

Reply:
example: https://i0.wp.com/filesamples.com/samples/image/jpeg/sample_640%C3%97426.jpeg

Reply:
Good catch @leogermani 👍 I'll add it. 

I don't think there are any extension variations on PNG or GIF.

Reply:
<img width="1422" alt="Screen Shot 2021-05-26 at 10 58 11" src="https://user-images.githubusercontent.com/17325/119578805-508be900-be11-11eb-894e-27ce20b9ec2d.png">

Now Photon-ized 👍